vowpalwabbit/cb_explore_adf.cc

```diff
@@ -90,31 +90,38 @@ template<class T> void swap(T& ele1, T& ele2)
   ele1 = temp;
 }
 
+// Validates a multiline example collection as a valid sequence for action dependent features format.
 example* test_adf_sequence(multi_ex& ec_seq)
 {
+  if (ec_seq.size() == 0)
+    THROW("cb_adf: At least line must be provided for an example to be valid.");
```

Comment:
We should use a semantic comment instead.  "At least one action must be provided for an example to be valid".